scripts/test

```diff
@@ -2,4 +2,4 @@
 # Specifying the project name (-p) creates a non-default network for the tests
 # to run isolated from the main app
 docker-compose -p test -f docker-compose-tests.yml down -v
-docker-compose -p test -f docker-compose-tests.yml up --build --force-recreate test
```

Comment:
I haven't encountered any problems since removing it so assume that it either wasn't needed or the problems we were seeing with bind mounted volumes have been resolved. Removing it make startup and tests much faster (obvs).